### PR TITLE
Import brightness

### PR DESCRIPTION
**Pull request by cquiroz**
Some suggested improvements to brightness import:
* Rename sys columns to unit
* trim magnitude values
* Support comment with lines starting with `#` or `//`
* Assign different default magnitudes to sloan bands

### CODE REVIEW

File: modules/catalog/src/main/scala/lucuma/catalog/csv/TargetImport.scala [hunk 1]
```diff
@@ -69,23 +69,32 @@ private case class TargetCsvRow(
...
   private lazy val integratedBrightness
     : List[(Band, Measure[BigDecimal] Of Brightness[Integrated])] =
-    brightnessAndUnit(brightnesses, integratedUnits, VegaMagnitudeIsIntegratedBrightnessUnit.unit)
+    brightnessAndUnit(
+      brightnesses,
+      integratedUnits,
+      b =>
+        if (b.tag.startsWith("Sloan")) ABMagnitudeIsIntegratedBrightnessUnit.unit
+        else VegaMagnitudeIsIntegratedBrightnessUnit.unit
+    )
```
Comment (rpiaggio) [inline modules/catalog/src/main/scala/lucuma/catalog/csv/TargetImport.scala:89]: `Band`s already have a default unit defined.

They can be accessed as, for example, `band.defaultIntegrated.units`

File: modules/catalog/src/main/scala/lucuma/catalog/csv/TargetImport.scala [hunk 2]
```diff
@@ -172,21 +181,37 @@ object TargetImport:
...
-  val surfaceUnits: Map[String, Units Of Brightness[Surface]] = unitAbbv[Surface]
+  val surfaceUnits: Map[String, Units Of Brightness[Surface]] = unitAbbv[Surface] ++ Map(
+    "Vega/arcsec2"    -> VegaMagnitudePerArcsec2IsSurfaceBrightnessUnit.unit,
+    "Vega/arcsec^2"   -> VegaMagnitudePerArcsec2IsSurfaceBrightnessUnit.unit,
+    "AB/arcsec2"      -> ABMagnitudePerArcsec2IsSurfaceBrightnessUnit.unit,
+    "AB/arcsec^2"     -> ABMagnitudePerArcsec2IsSurfaceBrightnessUnit.unit,
+    "Jy/arcsec2"      -> JanskyPerArcsec2IsSurfaceBrightnessUnit.unit,
+    "Jy/arcsec^2"     -> JanskyPerArcsec2IsSurfaceBrightnessUnit.unit,
+    "Jansky/arcsec2"  -> JanskyPerArcsec2IsSurfaceBrightnessUnit.unit,
+    "Jansky/arcsec^2" -> JanskyPerArcsec2IsSurfaceBrightnessUnit.unit
```
Comment (rpiaggio) [inline modules/catalog/src/main/scala/lucuma/catalog/csv/TargetImport.scala:199]: Can we derive these by replacing in the `abbv`? For example, whenever we have `²`, also add the keys replacing it with `2` and `^2`.

File: modules/catalog/src/main/scala/lucuma/catalog/csv/TargetImport.scala (hunk 1 above)
Comment (cquiroz) [reply modules/catalog/src/main/scala/lucuma/catalog/csv/TargetImport.scala:89]: Thanks, it is much better that way

File: modules/catalog/src/main/scala/lucuma/catalog/csv/TargetImport.scala [hunk 3]
```diff
@@ -170,23 +175,48 @@ object TargetImport:
...
+      "AB"     -> ABMagnitudeIsIntegratedBrightnessUnit.unit,
+      "Jy"     -> JanskyIsIntegratedBrightnessUnit.unit,
+      "Jansky" -> JanskyIsIntegratedBrightnessUnit.unit
+    ) ++ expandedAbbrevations[Integrated]
 
-  val surfaceUnits: Map[String, Units Of Brightness[Surface]] = unitAbbv[Surface]
+  val surfaceUnits: Map[String, Units Of Brightness[Surface]] =
+    unitAbbv[Surface] ++ expandedAbbrevations[Surface]
 
+  println(surfaceUnits)
```
Comment (toddburnside) [inline modules/catalog/src/main/scala/lucuma/catalog/csv/TargetImport.scala:206]: println still needed?

**Review by toddburnside — APPROVED** [review]
LGTM

File: modules/catalog/src/main/scala/lucuma/catalog/csv/TargetImport.scala (hunk 3 above)
Comment (cquiroz) [reply modules/catalog/src/main/scala/lucuma/catalog/csv/TargetImport.scala:206]: Nope, will remove it

**Pull request merged**